source/extensions/filters/http/adaptive_concurrency/config.cc

```diff
@@ -0,0 +1,42 @@
+#include "extensions/filters/http/adaptive_concurrency/config.h"
+
+#include "envoy/config/filter/http/adaptive_concurrency/v2alpha/adaptive_concurrency.pb.validate.h"
+#include "envoy/registry/registry.h"
+
+#include "common/config/filter_json.h"
+
+#include "extensions/filters/http/adaptive_concurrency/adaptive_concurrency_filter.h"
+#include "extensions/filters/http/adaptive_concurrency/concurrency_controller/noop_controller.h"
+
+namespace Envoy {
+namespace Extensions {
+namespace HttpFilters {
+namespace AdaptiveConcurrency {
+
+Http::FilterFactoryCb AdaptiveConcurrencyFilterFactory::createFilterFactoryFromProtoTyped(
```

Comment:
I don't think anything in this file or the NOP controller is actually used in this PR. Can we just remove it for now until we have the integration test?

Reply:
I'm actually forced to implement `createFilterFactoryFromProtoTyped` or the filter factory is an abstract class. I ripped out everything else I could.

Reply:
Sorry I meant I don't think you need config.h/config.cc at all in this PR? See https://github.com/envoyproxy/envoy/blob/master/source/extensions/extensions_build_config.bzl#L79 for an example of getting tests to run w/o a config file that we don't need yet.

/wait